pyiron_base/database/performance.py

```diff
@@ -0,0 +1,170 @@
+import pandas as pd
+from sqlalchemy import (
+    create_engine,
+    select,
+    distinct,
+    MetaData,
+    Table,
+    func,
+    or_,
+    false,
+)
+from pyiron_base.settings.generic import Settings
+
+
+__author__ = "Muhammad Hassani"
+__copyright__ = (
+    "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - "
+    "Computational Materials Design (CM) Department"
+)
+__version__ = "1.0"
+__maintainer__ = "Muhammad Hassani"
+__email__ = "hassani@mpie.de"
+
+
+class DatabaseStatistics:
+    def __init__(self):
+        s = Settings()
+        self._connection_string = s._configuration['sql_connection_string']
```

Comment:
This can be a local variable, as it is only used during the initialisation.

Reply:
Done!